### ISSUE TITLE
dev-tex/sagetex-3.6 recorded file size

### ISSUE THREAD

**Issue by hackitforfun** [cschwan/sage-on-gentoo#726]
I did a system update today on ~amd64, and find that emerge wants to downgrade dev-tex/sagetex from 3.6.1 to 3.6. This failed due to a downloaded file size mismatch, where the relevant part of the log is as follows.

```
!!! Fetched file: sagetex-3.6.tar.gz VERIFY FAILED!
!!! Reason: Filesize does not match recorded size
!!! Got:      69799
!!! Expected: 833486
```

**Comment by kiwifb** [comment]
That's bizarre, I only shipped sagetex 3.6.1 36 hours ago (it was hanging in an unfinished state for a couple of month in my tree). And did not touch it since. I probably shipped it with broken Manifest, I will check.

**Comment by kiwifb** [comment]
Right! 3.6 has SRC_URI from github which is not always stable. I'll switch it to pypi and recreate the Manifest.

**Comment by kiwifb** [comment]
Should be fixed by 1ee04f4bb26385b1e95d49cee46743b22642591d
